homeassistant/components/zwave_me/translations/en.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config": {
+        "step": {
+            "user": {
+                "title": "ZWaveMe Creation",
+                "description": "Input you WebSocket URL and Token here: TODO Where get token",
```

Comment:
No token handling currently done.

Reply:
Actually, we expect the user to copy it from our UI. We will eliminate the TODO and change the text.